cmd/olympus/actions/eventlog.go

```diff
@@ -0,0 +1,18 @@
+package actions
+
+import (
+	"fmt"
+
+	"github.com/gobuffalo/envy"
+	"github.com/gomods/athens/pkg/eventlog"
+	"github.com/gomods/athens/pkg/eventlog/mongo"
+)
+
+func newEventlog() (eventlog.Reader, error) {
+	mongoURI, err := envy.MustGet("OLYMPUS_MONGO_STORAGE_URL")
```

Comment:
Will mongo be the only one eventlog-store? I was thinking i will do this similar to newStorage() so it can be decided based on existence of a specific env var...